openedx/features/pakx/common/utils.py

```diff
@@ -0,0 +1,35 @@
+
+def load_space_in_session(request):
+    """
+    Set partner space in Session, extracts from query params or sets pakx as default space
+    """
+
+    space_param = request.GET.get("space", None)
+    active_space = request.session.get("space")
+    if active_space is None and space_param is None:
+        set_space_in_session(request, "pakx")  # Set Default space
+    elif space_param:
+        set_space_in_session(request, space_param)
```

Comment:
shouldn't we update the space if it's coming from query param regardless of active space? will this work?
```
        space_param = request.GET.get("space")
        set_space_in_session(request, space_param or "pakx")
```

Reply:
No, `"space_param or "pakx"`, the first request will have space, next one after login wouldn't have, so it will activate `pakx` which we don't want